lighthouse-plugin-publisher-ads/audits/bottleneck-requests.js

```diff
@@ -18,18 +18,20 @@ const {Audit} = require('lighthouse');
 const {computeAdRequestWaterfall} = require('../utils/graph');
 
 /** @typedef {LH.Artifacts.NetworkRequest} NetworkRequest */
-/** @typedef {LH.Gatherer.Simulation.NodeTiming} NodeTiming */
+/** @typedef {import('../utils/graph').SimpleRequest} SimpleRequest */
 
 const UIStrings = {
   title: 'No bottleneck requests found',
   failureTitle: 'Avoid bottleneck requests',
-  description: 'Speed up, parallelize, or eliminate the following ' +
-    'requests and their dependencies in order to speed up ad loading.',
+  description: 'Speed up, load earlier, parallelize, or eliminate the ' +
+    'following requests and their dependencies in order to speed up ad ' +
+    'loading.',
   displayValue: '{blockedTime, number, seconds} s spent blocked on requests',
   columnUrl: 'Blocking Request',
   columnInitiatorUrl: 'Initiator Request',
   columnStartTime: 'Start',
-  columnSelfTime: 'Self Time',
+  columnSelfTime: 'Exclusive Time',
```

Comment:
Key and value no longer match.
It would be nice if we had a tooltip type thing over the column names to explain what they mean...I don't think "self time" nor "exclusive time" is 100% clear...

Reply:
I think it's fine as-is. There are other locations in the code that mention "self time" and we may want to change the user-facing wording again in the future.

I'm happy to change this to something clearer, but I can't think of anything better ATM.

Reply:
SGTM